### ISSUE TITLE
Peer reviewer to make edits / comments to moderator guide

### ISSUE THREAD

**Issue by amejiamesinas** [hackforla/expunge-assist#933]
### Overview
We are finalizing our moderator guide to begin usability testing of the landing page. 

### Action Items
Please review the [Moderator Guide](https://docs.google.com/document/d/1RPihaS53bJnAF9oTFwglpQj_DUY2j260U0Xn_wNBTd0/edit), and add any comments/edits/suggestions. 
- Please complete this task before our next meeting on 5/19. 
- Once you have reviewed the moderator guide, please comment DONE. 
- If you are unable to review before our meeting, please comment, unable to review.

**Comment by SamHyler** [comment]
I think there's a lot to be done here, but the document is currently filled with comments and strike throughs so it is hard to follow. Who is the main cook of this kitchen @amejiamesinas ?

**Comment by sofialaguna** [comment]
done

**Comment by tntalcott** [comment]
done
